test/unit/objectLiteral.spec.ts

```diff
@@ -8,59 +8,53 @@ test.each([
     { inp: `{[myFunc()]:3,b:"4"}`, out: '{\n    [myFunc(_G)] = 3,\n    b = "4"\n}' },
     { inp: `{x}`, out: `{x = x}` },
 ])("Object Literal (%p)", ({ inp, out }) => {
-    const lua = util.transpileString(`const myvar = ${inp};`);
-    expect(lua).toBe(`local myvar = ${out}`);
+    const lua = util.testModule`const myvar = ${inp};`.getMainLuaCodeChunk();
+    expect(lua).toBe(`myvar = ${out}`);
```

Comment:
Maybe let's use snapshots there?

Reply:
I'd prefer to not use snapshots for this test.